ext/curl/tests/server.inc

```diff
@@ -3,7 +3,7 @@
 function curl_cli_server_start() {
     $php_executable = getenv('TEST_PHP_EXECUTABLE') ?: PHP_BINARY;
     $doc_root = __DIR__;
-    $router = "responder/get.inc";
+    $router = __DIR__ . "/responder/get.inc";
```

Comment:
This might be a behavioral change. It seems the tests should've also previously failed otherwise. I'll have to check.